### DISCUSSION TITLE
文档中的https://auto.gluon.ai/new/tutorials/multimodal/matching/text2text_matching.html教程，效果似乎不够好

### DISCUSSION THREAD

**Discussion by maguoyu2023** [autogluon/autogluon#2967] [Q&A]
我按照https://auto.gluon.ai/new/tutorials/multimodal/matching/text2text_matching.html
这里的代码运行了一边，并且把
from autogluon.multimodal import MultiModalPredictor

predictor = MultiModalPredictor(
        problem_type="text_similarity",
        query="premise", # the column name of the first sentence
        response="hypothesis", # the column name of the second sentence
        label="label", # the label column name
        match_label=1, # the label indicating that query and response have the same semantic meanings.
        eval_metric='auc', # the evaluation metric
    )

predictor.fit(
    train_data=snli_train,
    time_limit=180,
)
这里的time_limit=180改成了3600，但是运行3600s后，auc是挺高的达到了94%，macro_f1却很低，实际效果我添加到了csv文件中，请查看。
文件有6605行数据，label=1有3368，label=0有3237，但是预测的结果predict=1有6452，predict=0有153条，这说明1小时的训练，效果非常不好！
[test_res1.csv](https://github.com/autogluon/autogluon/files/10811650/test_res1.csv)

**Comment by zhome1226** [comment]
我也遇到了这个问题，我是f1值太低了，当我改成eval_metric='f1'时，模型的accuracy变化不大，f1值稍微升高了一点，但是还是不如人意{'f1': 0.46511627906976744,
 'accuracy': 0.9706382978723405,
 'balanced_accuracy': 0.6791790338538306,
 'mcc': 0.4697315592309085,
 'roc_auc': 0.7139227642276423,
 'precision': 0.6382978723404256,
 'recall': 0.36585365853658536}
